modules/access-object-collections.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: PROCEDURE
+[id="access-object-collections_{context}"]
+= Accessing deployment collections
+
+To use collections, click *Platform Configuration* -> *Collections*. The page displays a list of currently-configured collections. You can perform the following actions:
+
+* Search for collections by entering text in the *Search by name* field, and then press *->*. 
+* Click on a collection in the collection list to view the collection in read-only mode.
+* Click {kebab} for an existing collection to edit, clone, or delete it.
++
+[NOTE]
+====
+You cannot delete a collection that is actively used in {product-title-short}.
+====
+* Click *Create collection* to create a new collection.
```

Comment:
Does this create a new "Deployment Collection", as you describe in the next procedure right below this statement? Would it connect this bullet item to the text below if it said "... to create a new deployment collection"?